cloud/endagaweb/views/user.py

```diff
@@ -103,11 +106,28 @@ def auth_and_login(request):
     user = authenticate(username=request.POST['email'],
                         password=request.POST['password'])
     if user:
-        login(request, user)
-        next_url = '/dashboard'
-        if 'next' in request.POST and request.POST['next']:
-            next_url = request.POST['next']
-        return redirect(next_url)
+        if user.is_active:
+            login(request, user)
+            user = User.objects.get(username=user)
+            today = timezone.now()
+            user_profile = UserProfile.objects.get(user=user)
+            next_url = '/dashboard'
+            if 'next' in request.POST and request.POST['next']:
+                next_url = request.POST['next']
+            if (today - user_profile.last_pwd_update).days >= \
+                    settings.ENDAGA['PASSSWORD_EXPIRED_LAST_SEVEN_DAYS']:
+                text = str(user) + ' , your account will be blocked in next '\
```

Comment:
text should include words to the effect of '... unless you change your password.'

As a style point, I'd pretty much always use string formatting operator (`%`) rather than concatenating a bunch of strings.